tasks/quick_translation.rake

```diff
@@ -0,0 +1,70 @@
+# Usage : rake add_entry['fr','blacklight.search.my_new_command','Ma nouvelle commande']
```

Comment:
@j-dornbusch can you comment on the intended use of this rake task? Is this a proposed replacement for the process in https://github.com/samvera/hyrax/wiki/Translations-(internationalization)?

I'm not sure I understand.

Reply:
@no-reply Oops! I was not aware of the existence of the page you mention. Indeed, the rake task I added duplicates the existing process.

Reply:
@no-reply After checking, I think the rake task I added is an interesting solution for people who are not able or willing to put a card number on their Google Account. I remove it from the pull request to allow the merge. But we can share and document it if it is desired by the community.